ProofPoint/proofpoint-pod/ingest/parser.yml

```
@@ -118,6 +89,39 @@ stages:
       - set:
           tls.version: '{{json_event.message.tls.version}}'
         filter: '{{json_event.message.tls.version != "NONE"}}'
+  set_fields_for_message_parts:
+      - set:
+        event.category: ["email"]
+        event.dataset: 'msgPart'
+        event.type: ["info"]
+        email.local_id: '{{json_event.message.guid}}'
+        email.attachments: >
+            [
+              "file": {
+                "mime_type": "{{json_event.message.msgParts.detectedMime}}",
+                {% if json_event.message.msgParts.labeledName != "" %}
+                "name": "{{json_event.message.msgParts.labeledName}}",
+                {% else %}
+                "name": "{{json_event.message.msgParts.detectedName}}",
+                {% endif %}
+                {% if json_event.message.msgParts.labeledExt != "" %}
+                "extension": "{{json_event.message.msgParts.labeledExt}}",
+                {% else %}
+                "extension": "{{json_event.message.msgParts.detectedExt}}",
+                {% endif %}
+                "size": "{{json_event.message.msgParts.detectedSizeBytes}}",
+                "hash": {
+                  "md5": "{{json_event.message.msgParts.md5}}",
+                  "sha256": "{{json_event.message.msgParts.sha256}}",
+                }
+              }
+            ]
+        proofpoint.pod.urls: >
+            [
+              {% for url in json_event.message.msgParts.urls %}
+              "{{url.url}}",
```

To be coherent with the removed block.
```suggestion
              {% if url.url | is_url %}"{{url.url}}",{% endif %}
```